Lib/sqlite3/test/dump.py

```diff
@@ -1,7 +1,8 @@
 # Author: Paul Kippes <kippesp@gmail.com>
 
-import unittest
 import sqlite3 as sqlite
+import unittest
+
```

Comment:
Please remove any unrelated style changes from the PR. Ditto for added newlines (line 135, 144 and 149).